generic_config_updater/patch_sorter.py

```diff
@@ -1013,7 +1015,7 @@ def __init__(self, move_wrapper):
         self.move_wrapper = move_wrapper
         self.mem = {}
 
-    def rec(self, diff):
+    def sort(self, diff):
```

Comment:
small change not related to the PR.

The `MemoizationSorter` is not used so we have not caught this before. This sorter is just added to run experiments on the performance.